packaging/linux/Arch/sunshine.install

```diff
@@ -2,11 +2,18 @@ do_setcap() {
   setcap cap_sys_admin+p $(readlink -f $(which sunshine))
 }
 
+do_udev_reload() {
+  udevadm control --reload-rules
+  udevadm trigger --property-match=DEVNAME=/dev/uinput
+}
+
 post_install() {
   do_setcap
+  do_udev_reload
 }
 
 post_upgrade() {
   do_setcap
+  do_udev_reload
```

Comment:
can we put an echo here to tell user's they need to reboot?

Reply:
Turns out a `modprobe uinput` does the trick. See https://github.com/chrippa/ds4drv/issues/93